Makefile

```diff
@@ -195,6 +195,13 @@ bundle-update: ## Update containerImage, createdAt, and icon fields in the bundl
 bundle-reset-date: ## Reset bundle's createdAt
 	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ./bundle/manifests/$(OPERATOR_NAME)-operator.clusterserviceversion.yaml
 
+.PHONY: bundle-community
+bundle-community: bundle-k8s ## Update displayName, and description fields in the bundle's CSV
+	sed -r -i "s|displayName: Node Maintenance Operator|displayName: Node Maintenance Operator - Community Edition |;" ./bundle/manifests/$(OPERATOR_NAME)-operator.clusterserviceversion.yaml
+	sed -r -i "/This is a standalone.*/d;" ./bundle/manifests/$(OPERATOR_NAME)-operator.clusterserviceversion.yaml
```

Comment:
somehow this didn't get into the 1st review: I think we can delete these 2 line at all

Reply:
(this and the next)

Reply:
The text is removed in https://github.com/medik8s/node-maintenance-operator/pull/81.